src/GraphQL/Validation/ValidationOptions.cs

```diff
@@ -52,6 +52,9 @@ public ValidationOptions()
         /// </summary>
         public GraphQLOperationDefinition Operation { get; init; } = null!;
 
+        /// <inheritdoc cref="ExecutionOptions.RequestServices"/>
+        public IServiceProvider? RequestServices { get; init; } = null;
```

Comment:
I was a bit confused about changing struct size but it seems as a non-breaking change
https://stackoverflow.com/questions/37865610/is-changing-the-size-of-a-struct-a-breaking-change-in-c